flow360/error_messages.py

```diff
@@ -10,6 +10,12 @@
 """
 
 
+shared_submit_warning = """\
+You are submitting a resource from a shared account.
+To suppress this message run: flow360 configure --suppress-shared-account-prompt
+"""
```

Comment:
I think the message should read:
`You are submitting a resource to a shared account (email address).`

Reply:
Fixed in 5035ff0 - now confirmation works per-session and needs to be performed only once until the user logs out, either interactively or by the user calling 

```
fl.Accounts.confirm_submit()
```